UnityProject/Assets/Scripts/Items/Implants/ImplantBase.cs

```diff
@@ -87,17 +92,21 @@ private void Awake()
 			//If gendered part then set the sprite limb data to it
 			if (isDimorphic)
 			{
-				if(gender == Gender.Male)
-				{
-					limbSpriteData = maleSprite;
-				}
-				else if(gender == Gender.Female)
-				{
-					limbSpriteData = femaleSprite;
-				}
-				else
+				switch (gender)
```

Comment:
so the sprites are based on ``Gender`` instead of ``BodyType``? That's odd.

Reply:
I think it makes more sense to base it on body type. After what we wanted is separating body and identiy so people could have female body type and still be called with male pronoun, or whatever combination they prefer.

Reply:
should i then implement the ability to switch pronouns independently in char customization?

Reply:
I actually thought that was already a thing, but yes.